cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/AliasesTest.scala

```diff
@@ -192,20 +209,69 @@ class AliasesTest extends DocumentingTest with QueryStatisticsTestSupport {
+        query(
+          """CREATE ALIAS garden.flowers
+            |FOR DATABASE `perennial-flowers`""".stripMargin,
+          ResultAssertions(r => { assertStats(r, systemUpdates = 1) })
+        ) { statsOnlyResultTable() }
+        query(
+          """CREATE ALIAS garden.trees
+            |FOR DATABASE trees AT 'neo4j+s://location:7687'
+            |USER alice PASSWORD 'password'""".stripMargin,
+          ResultAssertions(r => { assertStats(r, systemUpdates = 1) })
+        ) { statsOnlyResultTable() }
+        p("When a database alias has been created in a composite database, it will show up in the `constituents` column provided by the command `SHOW DATABASES` and in the `SHOW ALIASES FOR DATABASE` command.")
+        query("SHOW DATABASE garden YIELD name, constituents", assertNameField("garden")) { resultTable() }
+        query("SHOW ALIASES FOR DATABASE WHERE name STARTS WITH 'garden'", assertNameField("garden.flowers", "garden.trees")) { resultTable() }
+        note {
+          p("Aliases cannot point to a composite database.")
+        }
+        // Query fails with `key not found: org.neo4j.cypher.docgen.tooling.ErrorOnlyTablePlaceHolder@4f1502d7`
+        // if it is placed inside the note :(
+        query("CREATE ALIAS yard FOR DATABASE garden", ErrorAssertions(error =>
+          error.getMessage should startWith("Failed to create the specified database alias 'yard': Database 'garden' is composite.")
+        )) { errorOnlyResultTable() }
+      }
     }
     section(title="Altering database aliases", id="alias-management-alter-database-alias", role = "enterprise-edition") {
+      initQueries(
+        "CREATE ALIAS `northwind` FOR DATABASE `northwind-graph-2020`",
+        """CREATE ALIAS `remote-northwind` FOR DATABASE `northwind-graph-2020` AT "neo4j+s://location:7687" USER alice PASSWORD 'password'""",
+        """CREATE ALIAS `remote-with-driver-settings` FOR DATABASE `northwind-graph-2020` AT "neo4j+s://location:7687" USER alice PASSWORD 'password'
+          |DRIVER { connection_timeout: duration({ minutes: 1 }), connection_pool_max_size: 10 }""".stripMargin,
+        """CREATE ALIAS garden.flowers
+          |FOR DATABASE `perennial-flowers`""".stripMargin,
+        """CREATE ALIAS garden.trees
+          |FOR DATABASE trees AT 'neo4j+s://location:7687'
+          |USER alice PASSWORD 'password'""".stripMargin
+      )
       p(
-        s"""Aliases can be altered using `ALTER ALIAS` to change its database target, url, user credentials, or driver settings.
+        s"""Aliases can be altered using `ALTER ALIAS` to change its database target, properties, url, user credentials, or driver settings.
            |The required privileges are described <<cypher-manual#access-control-dbms-administration-alias-management, here>>.
            |Only the clauses used will be altered.""".stripMargin)
       p("include::alias-management-syntax-alter-alias.asciidoc[]")
```

Comment:
```suggestion
      p("include::alias/alias-management-syntax-alter-alias.asciidoc[]")
```
This also applies to the create-alias syntax above

Reply:
this might also need to be updated in 4.4

Reply:
Fixed in 4.4 here: https://github.com/neo4j/neo4j-documentation/pull/1618